sdks/python/apache_beam/io/gcp/tests/pubsub_matcher_test.py

```diff
@@ -37,8 +37,15 @@
 except ImportError:
   pubsub = None
 
+try:
+  from google.protobuf.timestamp_pb2 import Timestamp
```

Comment:
Could this move into the same try statement as above? Do we expect protobuf and pubsub to be not installed at the same time?

Reply:
Ah I didn't realise you could have imports in the same try statement.  Fixing this up, which should also sort out the isort issues below (will handle it in the same way on pubsub_tests.py)